<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPanelItem</class>
 <widget class="QWidget" name="frmPanelItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="PanelItem" name="panelItem1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="PanelItem" name="panelItem2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="PanelItem" name="panelItem3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="PanelItem" name="panelItem4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cboxAlignment">
       <item>
        <property name="text">
         <string>左对齐</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>居中对齐</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>右对齐</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxColor">
       <item>
        <property name="text">
         <string>绿色</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>蓝色</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>水红色</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>黑色</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisable">
       <property name="text">
        <string>禁用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEnable">
       <property name="text">
        <string>启用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlarm">
       <property name="text">
        <string>报警</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNormal">
       <property name="text">
        <string>正常</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelItem</class>
   <extends>QWidget</extends>
   <header>panelitem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
